### DISCUSSION TITLE
Maintain version in topic of IRC channel

### DISCUSSION THREAD

**Discussion by midzer** [twbs/bootstrap#35185] [General] [answered]
Hi,
right now, Bootstrap's IRC channel on libera.chat is without any OP who can set BS version in topic properly. So I think this should be maintained to be up to date.

IIRC @bardiharborow set up permission when new channel was formed.

Have a nice day,
midzer

cc @XhmikosR

**Comment by XhmikosR** [comment]
@bardiharborow please add myself and @midzer as an OP

**Comment by midzer** [comment]
*bump*

**Comment by bardiharborow** [comment, marked as answer]
Hey folks. You did have access but it wasn't auto-opping when you joined. I've configured it to do so now, hopefully that works. :)

**Comment by midzer** [comment]
Seems like auto-opping isn't working for me any more. So I can't update topic.

**Comment by XhmikosR** [comment]
@bardiharborow can you have a look at this please?

**Reply by XhmikosR** [reply]
It seems auto-opping works for me AFAICT.
